Document/0x05b-Basic-Security_Testing.md

```diff
@@ -41,29 +41,50 @@ Note: On Linux, you need to choose an SDK directory. `/opt`, `/srv`, and `/usr/l
 
 #### Testing Device
 
+For dynamic analysis, you'll need an Android device to run the target app on. In principle, you can test without a real Android device and use only the emulator. However, apps execute quite slowly on the emulator, and this can make security testing tedious. Testing on a real device makes for a smoother process and a more realistic environment. On the other hand, emulators allow you to easily change SDK versions or create multiple devices. A full overview of the pros and cons of each approach is listed in the table below.
+
+| Property                   | Physical                                                                                                                                                                                                                                                     | Emulator                                                                                                                                              |
+|----------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Chance for bricking        | Soft bricked devices are always possible, but new firmware can typically still be flashed. Hard bricked devices are very rare.                                                                                                                                                   | Emulators can crash or become corrupt, but a new one can be created or a snapshot can be restored.                                                    |
```

Comment:
Chance for bricking sounds odd to me. But I don't see a good alternative immediately

Reply:
Maybe we call it "Ability to restore" instead?